config/config.go

```diff
@@ -1328,8 +1328,34 @@ func (c *Config) CheckConnectionMonitorConfig() {
 	}
 }
 
+// DefaultFilePath returns the default config file path
+// MacOS/Linux: $HOME/.gocryptotrader/config.json or config.dat
+// Windows: %APPDATA%\GoCryptoTrader\config.json or config.dat
+// Helpful for printing application usage
```

Comment:
It's very helpful when running an app with `--help` or `--usage` plus also has the added benefit of getting the default path without being as intensive as `config.GetFilePath`. All of our apps will now show the default config file path as a result of this PR instead of "" or failing if config.json doesn't exist in the default GCT dir

Reply:
I wasn't aware of `--usage`  :D Thanks for the explanation!